meteor/server/api/playout/playout.ts

```diff
@@ -364,6 +364,17 @@ export namespace ServerPlayoutAPI {
 			if (!nextPart) throw new Meteor.Error(404, `Part "${nextPartId}" not found!`)
 		}
 
+		// If we're setting the next point to somewhere other than the current segment, and in the queued segment, clear the queued segment
```

Comment:
A question, could this be simplified down to always clear if setManually is true? ie, if the user specified where the next is rather than sofie choosing

Reply:
I don't think we should clear the queued segment every time the user moves next, what if they just need to skip over the next part in the current Segment? e.g. An external feed has been lost, so they are skipping over an interview. If I was the user I'd want my queued selection to be kept in that scenario.